Underscore.cs/List/Implementation/Partition.cs

```diff
@@ -132,7 +140,23 @@ public IEnumerable<IEnumerable<T>> Chunk<T>( IList<T> list, Func<T, bool> on )
 			);
 	    }
 
-	    /// <summary>
+        /// <summary>
+        /// Takes a slice from a list, if start is greater then the end index
+        /// the results are reversed, if the index is negative corresponds to the index
+        /// from the back of the list
+        /// </summary>
+        /// <typeparam name="T">The type of the elements in the list</typeparam>
+        /// <param name="list">The list to take the slice from</param>
+        /// <param name="start">The start index</param>
+        /// <param name="end">The end index</param>
```

Comment:
is the end index exclusive/start index inclusive? mention it in the param comments